mix.exs

```diff
@@ -48,8 +48,8 @@ defmodule SignsUi.Mixfile do
       {:phoenix_html, "~> 2.10"},
       {:phoenix_live_reload, "~> 1.0", only: :dev},
       {:gettext, "~> 0.11"},
-      {:cowboy, "~> 1.0"},
-      {:plug_cowboy, "~> 1.0"},
+      {:cowboy, "~> 2.7"},
```

Comment:
I remembered something about this, so I went and checked, and it seems like explicitly depending on `cowboy` is not needed in general for Phoenix apps; the [1.4 upgrade guide](https://gist.github.com/chrismccord/bb1f8b136f5a9e4abc0bfc07b832257e) recommends removing it ("_replace_ your cowboy dependency with plug_cowboy").